SetCurrentVersion.cmd

```diff
@@ -1,9 +1,9 @@
 set MAJOR_PREVIOUS=1
-set MINOR_PREVIOUS=5
-set PATCH_PREVIOUS=46
+set MINOR_PREVIOUS=6
+set PATCH_PREVIOUS=0
 set PRERELEASE_PREVIOUS=
 
-set MAJOR=1
-set MINOR=6
+set MAJOR=2
+set MINOR=0
```

Comment:
Curious why this rates a major version change.

Reply:
Good question.  This probably should have been done with the original NET Core merge but I overlooked it.  I believe that was a big enough change to warrant a major version change.  What do you think?

---
In reply to: [145850825](https://github.com/Microsoft/sarif-sdk/pull/639#discussion_r145850825) [](ancestors = 145850825)

Reply:
Sure, let's do it now.